releases/2021-12-10-releasenotes.md

```diff
@@ -1,20 +1,38 @@
 ---
-title: '<Placeholder Title>'
-description: 'TBD'
+title: 'A Sleigh Full of New Features'
+description: 'Smart contracts, a pay-as-you-go pricing plan, the beta release of Azure support, and more!'
 date: '2021-12-10'
-draft: true
+draft: false
 tags:
 - New Features
-- Beta Features
-- Share Tiers
-- Share Pricing
-- Polycloud
 - Azure
+- Smart Contracts
+- Share Tiers and Pricing
+- ACID Semantics
 ---
 
-## Summary
-## Starter Tier
-### Starter Tier Limit Enforcement
-## Individual Tier
+The Vendia team has been hard at work building out new features that we know will bring cheer to our users.  We've been so busy and have so much to share that we've also created a [blog post](https://www.vendia.net/blog/announcing-new-features) to delve into more details about each new feature.
+
 ## Smart Contracts
-## Azure Support
\ No newline at end of file
+
+Vendia Share makes it easy to share data and files among participants in a Uni.  But sometimes _sharing_ data alone isn't enough.  Smart Contracts allow users to _take action_ on data in a prescribed way, as data changes across partners, clouds, and regions, without having to build or maintain a complex eventing infrastructure.
+
+![Smart Contracts](https://user-images.githubusercontent.com/92179243/145060082-56557805-7708-4a26-8818-8dcfb95ceb78.png)
+
+Please see the new [Smart Contracts docs](https://www.vendia.net/docs/share/smart-contracts) for more information on how to use this powerful new feature and for more information on how Vendia Smart Contracts compare with smart contracts in other solutions.
```

Comment:
This link still 404's. When are these links going live?

Reply:
Correct.  Waiting for Rory's PR to get merged for this one and then this link will resolve properly.